### PR TITLE
Prepare beta release of Key Vault SDKs

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 4.0.0-beta.4 (Unreleased)
+## 4.0.0-beta.4 (2021-02-10)
 
 ### Added
 
 - Support in `KeyVaultAccessControlClient` to create, update, and delete custom role definitions.
 
+### Changed
```
Comment (JoshLove-msft) [inline sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md:9]: I believe we are supposed to use "New Features" - https://azure.github.io/azure-sdk/policies_releasenotes.html

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md:9]: Meant to put this comment for Added, but Changed also doesn't seem like one of the allowed headers.

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md:9]: Thanks. That refers to release notes, however. This follows the current changelog guidance: https://azure.github.io/azure-sdk/policies_releases.html#2-change-logs--migration-guides

/cc @jsquire

Comment (JoshLove-msft) [reply sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md:9]: Ah ok looks like you are right.

Comment (jsquire) [reply sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md:9]: I'm not sure, to be honest.  Adrian had been the one that asked me to keep the changelog form in accordance to the release notes guidance.     

@AlexGhiondea,  @annelo-msft:   Do either of your folks have an authoritative answer on whether we're applying two different formats between change logs and release notes and, if not, which of these policies takes precedence?

**Pull request merged**